modules/eventing/pages/eventing-Terminologies.adoc

```diff
@@ -48,75 +67,93 @@ At times, the handler code can trigger data mutations. To avoid cyclic generatio
-handler code.
+An Eventing Function can have no binding, one binding, or several bindings.
 
-NOTE: Bindings are mandatory when your handler code performs any bucket related operations.
+*Bucket Bindings*
 
-Bindings ensure easy porting of source code across different environments. You can export-import Functions from test-to-test, test-to-production, or production-to-production environments, without making code changes.
+Bucket bindings allow JavaScript handlers to access Couchbase KV buckets. The buckets are then accessible by the bound name as a JavaScript map in the global space of the handler. 
 
-== Function Settings
+You can add bucket bindings via the 'bucket-alias' choice then entering a tuple of: alias-name, bucket-name, and an access level. Where the alias-name that you can use to refer to the bucket from your handler code; the bucket-name is the actual name of the bucket in the cluster; and the access level is either 'read only' or 'read and write'.
 
-The log levels, script timeout, and workers allocation are additional options available during the Function definition process.
+NOTE: One or more bucket bindings (or bucket-aliases) are mandatory when your handler code performs any bucket related operations.
 
-* *Log Levels*: Use log levels to determine the granularity of log messages.
-* *Script Timeout*: Script Timeout provides a timeout option to terminate a non-responsive Function.
-* *Workers*: Workers allocate the number of worker threads for a specific Function.
-+
-For a defined Function, Couchbase Server limits the maximum number of workers to 10.
-This upper limit is configured for system optimization purposes.
-In the Function definition, if the number of workers exceeds a set threshold, then the system automatically generates a warning message.
-However, this warning message does not prevent deployment of the Function.
+* Read Only Bindings: A binding with access level of "Read Only" allows reading documents from the bucket, but cannot be used to write (create, update or delete) documents in such a bucket. Attempting to do so will throw a runtime exception.
+
+* Read-Write Bindings: A binding with access level of "Read Write" allows both reading and writing (create, update, delete) of documents in the bucket.
+
+*URL Bindings*
+
+These bindings are utilized by the cURL language construct to access external resources. The binding specifies the endpoint, the protocol (http/https), and credentials if necessary. Cookie support can be enabled via the binding if desired when accessing trusted remote nodes. When a URL binding limits access through to be the URL specified or descendants of it. The target of a URL binding should not be a node that belongs to the Couchbase cluster.
+
+You can add URL bindings via the 'URL-alias' choice then entering the following: alias-name, URL, allow cookies setting, and security settings of validate SSL certificate and an auth type of (no auth, basic, bearer, and digest).
+
+= Operations
```

Comment:
This top-level heading (with one `=`) causes an error message when the docs are built:

```
level 0 sections can only be used when doctype is book
```

Please restructure this section to avoid this, e.g. change this heading to a sect1 heading (`==`) and all subsequent headings to sect2 headings (`===`).

Reply:
Resolving in PR#985